style.scss

```diff
@@ -0,0 +1,1263 @@
+        }
+        .about-us-content {
+            flex: 0 0 268px;
+            margin-right: 7.5625rem;
+            @media screen and (max-width:1280px) {
+                margin-right: 5.875rem;
+            }
+            @media screen and (max-width:1199px) {
+                margin-right: 5.625rem;
+                flex: 0 0 258px;
+            }
+            @media screen and (max-width:991px) {
+                margin-right: 4.125rem;
+                flex: 0 0 248px;
+            }
+            @media screen and (max-width:768px) {
+                flex: 0 0 238px;
+                margin-right: 2.875rem;
+            }
+            @media screen and (max-width:575px) {
+                width: 100%;
+                margin-right: 0;
+            }
+            .logo {
+                margin-bottom: 1.375rem;
+                @media screen and (max-width:376px) {
+                    margin-bottom: 1.1875rem;
+                    svg {
+                        width: 8.0625rem;
+                        height: 1.375rem;
+                    }
+                }
+            } 
+            p {
+                color: $textPrimary;
+                font-size: .875rem;
+                line-height: 22px;
+                max-width: 16.75rem;
+                margin-bottom: .125rem;
+                @media screen and (max-width:575px) {
+                    max-width: 100%;
+                }
+                @media screen and (max-width:376px) {
+                    display: none;
+                }
+            }
+            .mobile-content {
+                display: none;
+                @media screen and (max-width:376px) {
+                    display: block;
+                    font-size: .75rem;
+                    line-height: 18px;
+                    margin-bottom: 1rem;
+                }
+            }
+            .social-listing {
+                flex-direction: column;
+                @media screen and (max-width:376px) {
+                    flex-direction: row;
+                    justify-content: space-between;
+                    align-items: center;
+                }
+                .img-wrapper {
+                    @media screen and (max-width:376px) {
+                        max-width: 6.3125rem;
+                        max-height: 2.75rem;
+                        img {
+                            width: 100%;
+                            height: 100%;
+                        }
+                    }
+                }
+                .social-icons {
+                    margin: 1.5rem 0 2.75rem ;
+                    padding-left: 1.5625rem;
+                    .svg-wrapper:not(:last-child) {
+                        margin-right: .5rem;
+                    }
+                    @media screen and (max-width:376px) {
+                        padding-left: 0;
+                        margin: 0;
+                    }
+                }
+            }
+        }
+        .ul-wrapper {
```

Comment:
rename ul-wrapper 
like about-us-overview

Reply:
class name is changed into contact-us-overview